src/build.py

```diff
@@ -751,7 +751,10 @@ def GetRepoInfo():
 
 def OverrideCMakeCompiler():
   if IsWindows():
-    return []
+    MSVC_COMPILER = os.path.join(os.environ['GYP_MSVS_OVERRIDE_PATH'],
```

Comment:
Is this environment variable guaranteed to exist? If so, how; and if not, then we probably don't want to make the explicit override unconditional.